automationcommon/models.py

```diff
@@ -120,20 +120,35 @@ def _dict(self):
         # NOTE: an internal attribute has been used when introspecting the model.
         return model_to_dict(self, fields=[field.name for field in self._meta.fields])
 
+    def audit_compare(self, field, old, new):
```

Comment:
Could we not make this a method on the [Meta class](https://docs.djangoproject.com/en/2.0/topics/db/models/#meta-options)? Or, ideally, something like:
```python
class MyModel(ModelChangeMixin, models.Model)
    some_crazy_field = WeirdField()

    class Meta:
        def audit_compare_some_crazy_field(self, old, new):
            """WeirdField always stores the old value as a reversed list..."""
            return reverse(old) == new
```
where the field name is used to automatically find the right method.

If such automatic magicness is too much of a pain, I'd still prefer this to be on the Meta class.

Reply:
I was broadly led by this advise: https://code.djangoproject.com/ticket/5793

Reply:
OK